scripts/ci/post_install_hardening_hook.rb

```
@@ -0,0 +1,76 @@
+#!/usr/bin/env ruby
```

RuboCop (1.76.1)

[warning] Script file post_install_hardening_hook.rb doesn't have execute permission.

(Lint/ScriptPermission)